Gemfile.lock

```diff
@@ -0,0 +1,41 @@
+PATH
+  remote: .
+  specs:
+    payrix (0.1.2)
+      faraday (~> 0.14)
+      faraday_middleware (~> 0.12)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    diff-lcs (1.5.0)
+    faraday (0.17.5)
+      multipart-post (>= 1.2, < 3)
+    faraday_middleware (0.14.0)
+      faraday (>= 0.7.4, < 1.0)
+    multipart-post (2.2.0)
+    rake (10.5.0)
+    rspec (3.11.0)
+      rspec-core (~> 3.11.0)
+      rspec-expectations (~> 3.11.0)
+      rspec-mocks (~> 3.11.0)
+    rspec-core (3.11.0)
+      rspec-support (~> 3.11.0)
+    rspec-expectations (3.11.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-mocks (3.11.1)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-support (3.11.0)
+
+PLATFORMS
+  x86_64-darwin-19
+
+DEPENDENCIES
+  payrix!
+  rake (~> 10.0)
+  rspec
+
+BUNDLED WITH
+   2.3.10
```

Comment:
Since I can't comment on the `bin/console` file below, I'll comment on it here. I've updated the permissions on the file to allow it to be run in the command line, with `bin/console`.

I was able to do this though `chmod +x bin/console`. This command means change the access permissions to `bin/console` to include execution (`+x`).